csv-quiz-timer/quiz/quiz.go

```diff
@@ -0,0 +1,40 @@
+package quiz
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type Problem struct {
+	Question, Answer string
+}
+
+// Handles runtime errors
+func handleError(e error, message string) {
+	if e != nil {
+		fmt.Println(message)
+		os.Exit(1)
+	}
+}
+
+// Parses a csv file into a slice of Problem type
+func ParseFile(fileName string) []Problem {
+	f, err := os.Open(fileName)
+	handleError(err, fmt.Sprintf("Couldn't find file: %s\n", fileName))
```

Comment:
I would prefer to see error wrapping here using either https://golang.org/pkg/errors/#example_New_errorf or https://github.com/pkg/errors. We can talk about it.